assets/js/util/when-active.js

```diff
@@ -54,14 +55,14 @@ export default function whenActive( { moduleName, FallbackComponent = null, Inco
 			// eslint-disable-next-line react-hooks/rules-of-hooks
 			const module = useSelect( ( select ) => select( CORE_MODULES ).getModule( moduleName ) );
 
-			// Return null if the module is not loaded yet or doesn't exist.
+			// Return <WidgetNull /> if the module is not loaded yet or doesn't exist.
 			if ( ! module ) {
-				return null;
+				return <WidgetNull widgetSlug={ moduleName } />;
```

Comment:
`widgetSlug` is never needed by the component which is passed via props since we pre-provide and "lock" this in the given component already. We have no way of knowing what the widget slug is anyways.
```suggestion
				return <WidgetNull />;
```